src/schema/types.ts

```diff
@@ -147,3 +148,24 @@ export type DataTrackedAttrs = {
   userID: string
   createdAt: number
 }
+
+export type SectionCategory = {
+  _id: string
+  name: string
+  synonyms: string[]
+  isEditable: boolean
+  isDisabled?: boolean
+  titles?: string[]
```

Comment:
Can we really have a category without a title?

Reply:
I'm using the titles property for auto-complete (which only a few sections use). I had to differentiate between titles and synonyms because the old titles we had weren't the ones used by the autocomplete but were used for the import and I didn't think we want to support all previous titles for autocomplete.

Reply:
If we get rid of the name, then can probably have titles as required and use the first title as the name.